components/TransLine.tsx

```diff
@@ -28,6 +28,16 @@ function resolveUrl(link: I18nString, userArrivedFromUioMobile: boolean) {
   return t(link)
 }
 
+/**
+ * Determine if this link is interal to UIO for our purposes
```

Comment:
```suggestion
 * "Internal" UIO/BPO links should not open in a new tab, vs "external" links to EDD, which should.
```

Was initially correcting a typo, here it is if you don't like my change:
```suggestion
 * Determine if this link is internal to UIO for our purposes
```